.github/workflows/sync-labels.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     name: Sync labels
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
```

Comment:
I don't think this file is controlled by us. Are we supposed to update this on a repo level?

Reply:
Ehm, not really, this is more like collateral damage. It doesn't hurt 🤷